axelrod/result_set.py

```diff
@@ -94,12 +101,12 @@ def build_match_lengths(self):
 
         for rep in range(self.nrepetitions):
 
-            for player_pair_index, match in self.matches[rep].items():
+            for player_pair_index, interactions in self.interactions[rep].items():
                 i, j = player_pair_index
```

Comment:
I think I've got all these. Good be worth checking I understood correctly (c26bf8fab4e5b7ce59c146de9c5c20073c8835b3).